src/grpc/utils.hpp

```diff
@@ -29,4 +29,36 @@ static bool get_file_contents(const std::string& file_name, std::string& content
     return false;
 }
 
-} // namespace sisl::grpc
+[[maybe_unused]] static void serialize_to_byte_buffer(io_blob_list_t const& cli_buf, grpc::ByteBuffer& cli_byte_buf) {
```

Comment:
maybe unused is for the function, not the return value. Since these are free functions, if another file includes this and not use the function, compiler gives ununsed function error